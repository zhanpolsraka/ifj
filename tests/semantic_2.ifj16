//3
class Main
{
	static void run()
	{
		int x;
		x = 3;
		return x;
	}
}
