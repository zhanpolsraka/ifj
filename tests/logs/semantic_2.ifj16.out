
Semantic error! Expected return value, but function returns value with other type or return nothing!

